docs/Internationalization.md

```diff
@@ -16,5 +16,11 @@ or
 
     LOCALE=fr yarn dev
 
+To host a localized version of Pinafore using Vercel, in the [vercel.json file](https://github.com/nolanlawson/pinafore/blob/master/vercel.json), replace the buildCommand value accordingly.
```

Comment:
Rather than say "replace the buildCommand value accordingly," I think it would be more helpful to refer directly to the line of code:

https://github.com/nvdaes/vercelPinafore/blob/45c70fb2088fe5f2380a729dab83e6f3ab4e6291/vercel.json#L9

Otherwise it's not necessarily clear what needs to be done.